app/src/validators/bucket.js

```diff
@@ -36,6 +36,7 @@ const schema = {
 
   searchBuckets: {
     query: Joi.object({
+      userId: scheme.guid,
```

Comment:
For now, if this is a singular GUID, IIRC we should be using the uuidv4 scheme instead of scheme.guid, which permits one, or an array of guids.